src/callbacks.c

```diff
@@ -307,16 +307,15 @@ G_MODULE_EXPORT void on_redo1_activate(GtkMenuItem *menuitem, gpointer user_data
 
 G_MODULE_EXPORT void on_cut1_activate(GtkMenuItem *menuitem, gpointer user_data)
 {
-	GeanyDocument *doc = document_get_current();
 	GtkWidget *focusw = gtk_window_get_focus(GTK_WINDOW(main_widgets.window));
 
-	if (GTK_IS_EDITABLE(focusw))
+	if (GTK_IS_EDITABLE(focusw)) {
```

Comment:
Sorry about that, there was some ambiguity i.e. 
https://github.com/geany/geany/blob/master/src/callbacks.c#L279 vs https://github.com/geany/geany/blob/master/src/callbacks.c#L235. Do you prefer single-line if statements to be non-nested in general?